apps/plato_single_variant/Readme.md

```diff
@@ -24,26 +21,26 @@ There are two input options required to run the app:
     | 4 |   4 |   1   |   5.3    |
 
 Other optional inputs:
-- Continuous Covariates: A tab-separated file with columns: `FID`,`IID`,`Cov1`,`Cov2`,`....`
-Here values of the covariate should be numeric and quantitative.
-- Categorical Covariates: A tab-separated file with columns: `FID`,`IID`,`Cat_Cov1`,`Cat_Cov2`,`....`.
-Here entry of non-numeric classes are allowed and will automatically generate a dummy encoding.
-- Sample File: User-specified list of sample ID's to include in the analysis. The file format should be a space/tab-delimited text file with family IDs (`FID`) in the first column and individual IDs (`IID`) in the second column.
-- SNP file: User-specified list of SNPs to include in the analysis. The file format should be either one `RSID` per line or positions in rage format (`chr`,`start`,`stop`,`label`)
+  - Continuous Covariates: A tab-separated file with columns: `FID`,`IID`,`Cov1`,`Cov2`,`....`
+    - Here values of the covariate should be numerical continuous.
+  - Categorical Covariates: A tab-separated file with columns: `FID`,`IID`,`Cat_Cov1`,`Cat_Cov2`,`....`.
+    - Here entry of non-numeric classes are allowed and will automatically generate a dummy encoding.
+  - Sample File: User-specified list of sample ID's to include in the analysis. The file format should be a space/tab-delimited text file with family IDs (`FID`) in the first column and individual IDs (`IID`) in the second column.
+  - SNP file: User-specified list of SNPs to include in the analysis. The file format should be either one `RSID` per line or positions in rage format (`chr`,`start`,`stop`,`label`)
 
-### Options
+## Options
 
 ### REGRESSION OPTIONS
 **Regression**:
 PLATO app can perform various kind of regression methods:
--   **Regress-auto** : It is default regression method for the app. It will perform either a linear or logistic regression based on the phenotype provided to the app. It works well when you have quantitative and categorical outcomes in the input phenotype file.
--   **Linear**: This option performs a simple (ordinary least-squares) regression between SNP and outcome/s.
--   **Logistic**: This option performs logistic regression between SNP and outcome/s. The logistic regression is performed using a Newton-Rhapson iteratively reweighted least squares algorithm.
--   **Firth**:  This option perform Firth bias reduction to the models being tested (for details see Firth, 1993 and Heinze and Shempel, 2002). This option can be helpful when using many categorical covariates as control variables. Also, this can be helpful when testing a large number of dichotomous phenotypes with widely varying case/control ratios, as you may see in a PheWAS.
+  - **Regress-auto** : It is default regression method for the app. It will perform either a linear or logistic regression based on the phenotype provided to the app. It works well when you have quantitative and categorical outcomes in the input phenotype file.
```

Comment:
- **Regress-auto** : The default regression method for the app. It will perform either a linear or logistic regression based on the phenotype provided to the app. It works well when you have quantitative and categorical outcomes in the input phenotype file.